exporter/kafkaexporter/README.md

```diff
@@ -21,7 +21,9 @@ The following settings can be optionally configured:
   - `otlp_json`:  ** EXPERIMENTAL ** payload is JSON serialized from `ExportTraceServiceRequest` if set as a traces exporter or `ExportMetricsServiceRequest` for metrics or `ExportLogsServiceRequest` for logs. 
   - The following encodings are valid *only* for **traces**.
     - `jaeger_proto`: the payload is serialized to a single Jaeger proto `Span`, and keyed by TraceID.
-    - `jaeger_json`: the payload is serialized to a single Jaeger JSON Span using `jsonpb`, and keyed by TraceID.
+    - `jaeger_json`: the payload is serialized to a single Jaeger JSON Span using `jsonpb`, and keyed by TraceID.\
+  - The following encodings are valid *only* for **logs**.
+    - `raw`: if the log record body is a byte array, it is sent as is. Otherwise, it is serialized to JSON.
```

Comment:
I feel like we should move the encoding out into a seperate doc at this point.

I also think it is worthwhile mentioned that we can't infer any of the metadata like attributes or resources using raw.

Reply:
not sure where to move things. I'll add a sentence regarding metadata.

Reply:
This might come after this - I might ask that we move encoders to a lib and reuse them across multiple receivers/exporters. Useful for my work on event hubs for Azure.